pysb/bng.py

```diff
@@ -116,6 +119,11 @@ def __enter__(self):
     def __exit__(self):
         return
 
+    def _delete_tmpdir(self):
+        for f in os.listdir(self.base_directory):
+            os.unlink(os.path.join(self.base_directory, f))
+        os.rmdir(self.base_directory)
+
```

Comment:
Thought about it - it seemed safer to do it this way rather than `shutil.rmtree` as we don't want to delete recursively (with the current setup, anyway). I guess that may change and `rmtree` is cleaner though. I'll make the change.